### PR TITLE
Default host is now ours.

### PR DESCRIPTION
**Pull request by grepme**

### CODE REVIEW

**Comment by grepme** [comment]
I've tested this out on a few levels, it should close #67

File: social_network/settings.py [hunk 1]
```diff
@@ -117,3 +117,11 @@
...
 MEDIA_ROOT = os.path.join(BASE_DIR, 'uploads')
 MEDIA_URL = '/uploads/'
+
+#
+# CUSTOM SETTINGS
+# These are application specific and not for Django but rather the underlining application
+# As such, they need to be prefixed with CUSTOM_ to prevent collision
+#
+
+CUSTOM_HOST_DEFAULT = "projecthub.ca"
```
Comment (varsis) [inline social_network/settings.py]: Should be http://projecthub.ca/ to match spec.

**Comment by JakeBrand** [comment]
Much better than hardcoding, but if we need the http:// then I'd wait until that is included before merging.

**Comment by grepme** [comment]
Okay, fixed. Please merge if good.

**Pull request merged**